rebasehelper/cli.py

```diff
@@ -230,6 +230,12 @@ def build_parser():
             default=None,
             help="new upstream sources"
         )
+        parser.add_argument(
+            "--changelog-entry",
+            default="- New upstream release %{version}",
+            help="format shown in changelog after updating to a higher version, "
+                 "you can use RPM macros (%{version} etc.)"
```

Comment:
I think something like this sounds better (and is more consistent with help text of other options): `text to use as changelog entry, can contain RPM macros, which will be expanded`.